cdap-app-fabric/src/test/java/co/cask/cdap/internal/guice/AppFabricTestModule.java

```diff
@@ -131,6 +131,7 @@ protected void configure() {
     // we want to use RemotePrivilegesFetcher in this module, since app fabric service is started
     install(new AuthorizationEnforcementModule().getStandaloneModules());
     install(new SecureStoreModules().getInMemoryModules());
+    install(new PreviewServerModule());
```

Comment:
Creating injector in `PreviewHttpHandlerTest` and not extending from `AppFabricTestBase`.